great_expectations/data_context/data_context/context_factory.py

```diff
@@ -457,6 +460,21 @@ def get_context(
 ) -> FileDataContext: ...
 
 
+@overload
+def get_context(
+    project_config: DataContextConfig | Mapping | None = ...,
+    context_root_dir: PathStr = ...,  # If context_root_dir is provided, project_root_dir shouldn't be  # noqa: E501 # FIXME CoP
```

Comment:
This signature says `context_root_dir` is non-`None` and `project_root_dir` is `None` and the other signature has these flipped. Do we enforce that anywhere, eg raise if both are non-None?
I'm just wondering if we don't enforce these, should we combine signatures. That is, is it now ok and safe to pass in both?

Reply:
No, I've seen errors raised if you pass both